content_schemas/formats/shared/base_links.jsonnet

```diff
@@ -1,22 +1,21 @@
 {
```

Comment:
https://github.com/alphagov/publishing-api/pull/2669/commits/d85bfbfb4212073e09106c4eaf318ba4ba28ae13 is a big commit and hard to review. I've always been slightly confused by the difference between base_links and whitehall_edition_links. What do you think about splitting this commit:
- Remove topic from `base_links`, and run `rake build_schemas` as per the [docs](https://github.com/alphagov/publishing-api/blob/main/docs/content_schemas/changing-an-existing-content-schema.md#general-workflow). What I don't think that will do is update the examples so you'd have to add those to the commit manually. Perhaps by running `rake` and seeing which fail?
- Remove topic from `whitehall_edition_links` and same as above.

Reply:
I will split it later before putting this PR as ready for review.

Reply:
I've separated dist files (they are autogenerated) from others, which should make it easier to review.